packages/react-native/services/mongodb-remote/__tests__/RemoteMongoClientIntTests.ts

```diff
@@ -749,4 +753,262 @@ describe("RemoteMongoClient", () => {
 
     expect(coll.findOne()).rejects.toBeTruthy();
   });
+
+   /* tslint:disable:no-string-literal */
+   it("should properly support watch streams on set of ids", async () => {
```

Comment:
I think you should split this up into smaller tests personally. This is essentially just 4 tests put into one test

Reply:
Discussed offline, these tests are basically just copy-pasted from the `watch` tests in the other two SDKs. We're going to leave as is.